pyproject.toml

```diff
@@ -61,7 +61,6 @@ dependencies = [
   "pooch",
   "pyproj>=3.3.0",
   "requests>=2.0",
-  "roocs_grids>=0.1.2",
```

Comment:
`roocs-grids` is still a requirement, right ?

Reply:
Yes, removing it was just a misconception on my part. I added it back in the next commit.